src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -52,7 +52,7 @@ var adyenHelperObj = {
   LOADING_CONTEXT_LIVE:
     'https://checkoutshopper-live.adyen.com/checkoutshopper/',
 
-  CHECKOUT_COMPONENT_VERSION: '3.18.2',
+  CHECKOUT_COMPONENT_VERSION: '4.5.0',
```

Comment:
Is this also what got update in the bundle? Have you confirmed this does not break other parts of the checkout process through Testcafe?